src/uu/uniq/src/uniq.rs

```diff
@@ -388,12 +431,41 @@ pub fn uu_app() -> Command {
                 .help("end lines with 0 byte, not newline")
                 .action(ArgAction::SetTrue),
         )
+        .group(
+            // in GNU `uniq` every every digit of these arguments
+            // would be interpreted as a simple flag,
+            // these flags then are concatenated to get
+            // the number of fields to skip.
+            // in this way `uniq -1 -z -2` would be
+            // equal to `uniq -12 -q`, since this behavior
+            // is counterintuitive and it's hard to do in clap
+            // we handle it more like GNU `fold`: we have a flag
+            // for each possible initial digit, that takes the
+            // rest of the value as argument.
+            // we disallow explicitly multiple occurrences
+            // because then it would have a different behavior
+            // from GNU
+            ArgGroup::new(options::OBSOLETE_SKIP_FIELDS)
+            .multiple(false)
+            .args(OBSOLETE_SKIP_FIELDS_DIGITS)
```

Comment:
Rustfmt is not fixing this due to a long string somewhere else, but these lines would be indented.